src/Compilers/CSharp/Test/Emit/CSharpCompilerEmitTest.csproj

```diff
@@ -55,9 +56,6 @@
     <Reference Include="..\..\..\..\..\packages\System.Reflection.Metadata.$(SystemReflectionMetadataVersion)\lib\portable-net45+win8\System.Reflection.Metadata.dll" />
     <Reference Include="..\..\..\..\..\packages\System.Collections.Immutable.$(SystemCollectionsImmutableVersion)\lib\portable-net45+win8+wp8+wpa81\System.Collections.Immutable.dll" />
     <Reference Include="..\..\..\..\..\packages\Microsoft.DiaSymReader.1.0.5\lib\net45\Microsoft.DiaSymReader.dll" />
-    <Reference Include="xunit">
-      <HintPath>..\..\..\..\..\packages\xunit.1.9.2\lib\net20\xunit.dll</HintPath>
-    </Reference>
```

Comment:
@jaredpar most will only need to reference xunit.core and xunit.assert, but something will need to copy xunit.abstractions as well.

Naïve question--what's the issue with having multiple Imports from the build folder?  Were we trying to avoid transitively importing the same thing twice?